data_model/src/asset.rs

```diff
@@ -158,6 +158,7 @@ mod model {
         Debug, Display, Clone, IdEqOrdHash, Decode, Encode, Deserialize, Serialize, IntoSchema,
     )]
     #[display(fmt = "{id} {mintable}{type_}")]
+    #[serde(rename = "AssetDefinition")]
```

Comment:
This renaming look a bit inconsistent with schema, because in schema type `NewAssetDefinition`  still exist.

Reply:
Imo that's OK. This only affects json serialization, but type should still be different